docs/book/src/topics/mover.md

```diff
@@ -0,0 +1,163 @@
+kube-system                         coredns-66bff467f8-ggn54                                         1/1     Running   0          40m
+kube-system                         coredns-66bff467f8-t4bqr                                         1/1     Running   0          40m
+kube-system                         etcd-kind-control-plane                                          1/1     Running   1          40m
+kube-system                         kindnet-ng2gf                                                    1/1     Running   0          40m
+kube-system                         kube-apiserver-kind-control-plane                                1/1     Running   1          40m
+kube-system                         kube-controller-manager-kind-control-plane                       1/1     Running   1          40m
+kube-system                         kube-proxy-h6rmz                                                 1/1     Running   0          40m
+kube-system                         kube-scheduler-kind-control-plane                                1/1     Running   1          40m
+local-path-storage                  local-path-provisioner-bd4bb6b75-ft7wh                           1/1     Running   0          40m
+```
+
+Target cluster (Below is an example of external cloud provider)
+```
+# kubectl get pods --kubeconfig capi-openstack-3.kubeconfig --all-namespaces
+NAMESPACE     NAME                                                         READY   STATUS    RESTARTS   AGE
+kube-system   calico-kube-controllers-59b699859f-djqd7                     1/1     Running   0          6m2s
+kube-system   calico-node-szp44                                            1/1     Running   0          6m2s
+kube-system   calico-node-xhgzr                                            1/1     Running   0          6m2s
+kube-system   coredns-6955765f44-wk2vq                                     1/1     Running   0          21m
+kube-system   coredns-6955765f44-zhps9                                     1/1     Running   0          21m
+kube-system   etcd-capi-openstack-control-plane-82xck                      1/1     Running   0          22m
+kube-system   kube-apiserver-capi-openstack-control-plane-82xck            1/1     Running   0          22m
+kube-system   kube-controller-manager-capi-openstack-control-plane-82xck   1/1     Running   2          22m
+kube-system   kube-proxy-4f9k8                                             1/1     Running   0          21m
+kube-system   kube-proxy-gjd55                                             1/1     Running   0          21m
+kube-system   kube-scheduler-capi-openstack-control-plane-82xck            1/1     Running   2          22m
+kube-system   openstack-cloud-controller-manager-z9jtc                     1/1     Running   1          4m9s
+```
+
+# Install OpenStack Cluster API provider into target cluster
+
+You need install OpenStack cluster api providers into `target` cluster first.
```

Comment:
I know this doc has been there before. 

I would rather not duplicate the documentation of the main book.

Reply:
the main book is about `bootstrap` cluster,
here is the `workload` cluster...